packages/backend/src/storage.ts

```diff
@@ -62,21 +66,38 @@ export default (
     });
 
     router.post('/new/:ensName/addMessage', async (req, res, next) => {
-        const { encryptedEnvelopContainer, encryptedContactName, messageId } =
-            req.body;
+        const {
+            encryptedEnvelopContainer,
+            encryptedContactName,
+            messageId,
+            createdAt,
+        } = req.body;
 
-        if (!encryptedEnvelopContainer || !encryptedContactName || !messageId) {
+        if (
```

Comment:
any reason not to use the schema validator tool here?

Reply:
Until now we only used the schema validation tools for the lib function. However, I think it'll improve our code quality a lot if we use it on the backend as well.

I'll add schema generation similar to the lib package to the storage endpoints